lib/src/emoji.dart

```diff
@@ -51,38 +53,38 @@ class Emoji {
   static const variationSelector16 = 65039;
   static const ZWJ = 8205;
 
-  final String name;
-  final String char;
-  final String shortName;
-  final EmojiGroup emojiGroup;
-  final EmojiSubgroup emojiSubgroup;
+  final String? name;
```

Comment:
Should these fields of Emoji even by nullable?
I would assume that every Emoji has a name, char and shortName, so I think it would be better if we made those fields non-nullable. That would eliminate a lot of null checks and null casts down the line.

Reply:
The user could use type in an invalid name, resulting in that emoji not being found. Maybe some of the fields used internally can be non-nullable. Currently, I'm stumped on time, so if anyone could take a look at it and make a PR, it'll be much appretiated